orchestrator/orchestrator_test.go

```diff
@@ -20,45 +17,121 @@ type TestSuite struct {
 	orchestrator *Orchestrator
 }
 
-func createTestSuite(t *testing.T) *TestSuite {
+type TestCase struct {
+	name    string
+	l1Host  string
+	l2Host  string
+	wantErr bool
+}
+
```

Comment:
to remove - see comment below - these can just be individual tests that don't use createTestSuite